playbook/app/pb_kits/playbook/pb_text_input/_text_input.tsx

```diff
@@ -104,34 +103,38 @@ const TextInput = (props: TextInputProps, ref: React.ElementRef<"input">) => {
     <React.Fragment>
       <Flex
           className={`add-on-${addOnAlignment} ${borderCss}`}
-          inline="flex-container"
+          inline
```

Comment:
Wondering why this had to be removed here.

Reply:
The `inline` prop on Flex is a boolean so I wasn't able to make Typescript compile without changing this. I assumed that everything different from false is being understood as `true` but I wasn't table to put `true` doe to this rule:

https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md